slf4j/BUILD

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+# Author: ekempin@google.com (Edwin Kempin)
```

Comment:
Let's delete this line - author lines aren't helpful to begin with. 

Actually, the line above also isn't necessary for build files